src/Chip.js

```diff
@@ -12,11 +12,11 @@ const Chip = ({ children, close }) => {
 
 Chip.propTypes = {
   children: PropTypes.node,
-  /**
-   * If show a close icon
```

Comment:
I think we should keep the description (maybe a rewording)

Reply:
`Show a close icon`

Reply:
Ok, will add that